modules/tax/tests/src/TaxBrowserTestBase.php

```diff
@@ -1,13 +1,13 @@
 <?php
 
-namespace Drupal\commerce_tax\Tests;
+namespace Drupal\Tests\commerce_tax;
```

Comment:
All PHPUnit based tests go in Drupal\Tests\<module\Unit|Kernel|Functional|FunctionalJavascript